bin/run_functions.js

```diff
@@ -454,13 +458,16 @@ function executeProject(filename, project, options) {
                     process.exit();
                 }
             } else {
-                if (!('async' in options)) {
+                if (!options.async || !('async' in options)) {
                     // If status isn't CANCELED, PENDING or DISCONNECTED and we have an output directory, store reports there
                     //
                     if (!missingFiles) {
                         if (config.showProgress)
                             util.output('');
                         util.output("Result: " + status);
+                        if (options.printReport) {
```

Comment:
Is this running another call to the server? I think you get the report as the response text when running async.

Reply:
Oh yes this is another call to server.You are right, response can be used as the report. 👍